pandas/core/sparse/frame.py

```diff
@@ -242,6 +242,11 @@ def _init_spmatrix(self, data, index, columns, dtype=None,
     def to_coo(self):
         return SparseFrameAccessor(self).to_coo()
 
+    def __repr__(self):
+        with warnings.catch_warnings():
```

Comment:
why are you adding this?

Reply:
This wasn't added by me. It turned up in this PR after I did a pull from upstream towards the beginning of this PR. This is there in the upstream master branch.

Reply:
@intEll1gent can you do a 

```
git fetch upstream master
git merge upstream/master
```

in your branch, and then push again? 
That should make sure that those changes are not visible here.